vgmdb_importer.user.js

```diff
@@ -2,7 +2,7 @@
 // @name           Import VGMdb releases into MusicBrainz
 // @namespace      https://github.com/murdos/musicbrainz-userscripts/
 // @description    One-click importing of releases from vgmdb.net into MusicBrainz
-// @version        2021.7.13.1
+// @version        2021.11.12.1
 // @downloadURL    https://raw.githubusercontent.com/murdos/musicbrainz-userscripts/master/vgmdb_importer.user.js
 // @updateURL      https://raw.githubusercontent.com/murdos/musicbrainz-userscripts/master/vgmdb_importer.user.js
 // @include        /^https://vgmdb.net/(album|artist|org)/\d+
```

Comment:
Thanks, the fix for the missing `Commercial` attribute itself looks good to me, but I noticed that the regex of the include rule is broken:
```suggestion
// @include        /^https://vgmdb.net/(album|artist|org)/\d+/
```
Not sure how this has been working before, since the terminating slash had been missing right from the start.
But I could swear that the userscript did also work for me (Firefox/Violentmonkey) before... Maybe I had fixed that locally and forgot to submit a PR?

Reply:
Huh, I guess that's right: https://wiki.greasespot.net/Include_and_exclude_rules

I don't think I every used this script with anything other than Tampermonkey. Did it stop working in Violentmonkey when you tried this patch out?

Reply:
Yes, it did work for two of the three example releases from https://github.com/murdos/musicbrainz-userscripts/issues/404#issuecomment-927127335. Then I installed the patched version from this PR and it was no longer being injected at all, that's why I guessed that I probably had fixed my local installation.

Reply:
Makes sense, thanks for the fix!